static/monorail-2.0.yaml

```diff
@@ -221,6 +247,24 @@ definitions:
             type: object
         type: object
     type: object
+  relations.2.0_Relation:
+    description: A key value set of relations.
+    properties:
+      containedBy:
```

Comment:
Seems our code only enables the containedBy/contains relation at present, so I am OK to keep not adding other relations in your schema.